config/crd/default/bases/dynatrace.com_dynakubes.yaml

```diff
@@ -996,8 +996,7 @@ spec:
                   ActiveGateSpec `json:"activeGate,omitempty"`
                 properties:
                   capabilities:
-                    description: Activegate capabilities enabled (routing, kubernetes-monitoring,
-                      data-ingest)
+                    description: Activegate capabilities enabled (routing, kubernetes-monitoring)
```

Comment:
this is auto-generated by running `make manifests`, you'd have to remove it from `activegate_types.go`